hooks/boot/03-foreman-maintain-extensions.rb

```diff
@@ -1,6 +1,3 @@
-require 'English'
```

Comment:
the changes here are obviously not required, but it felt better to unify the code.